app/src/lib/editors/win32.ts

```diff
@@ -220,6 +220,13 @@ const editors: WindowsExternalEditor[] = [
     displayNamePrefix: 'VSCodium',
     publisher: 'Microsoft Corporation',
   },
+  {
+    name: 'Microsoft Visual Studio Community 2017',
+    registryKeys: [Wow64LocalMachineUninstallKey('71fbddf9')],
```

Comment:
Hello @devsebastian 

A few months ago I tried to add VSC support to GitHub desktop, but the registry key was an issue. It looks like Microsoft uses unstable hash as a key for the product. As far as I remember the keys may differ for each installation, even on a single machine.

The key for Microsoft Visual Studio Community 2017 is: `3f7d1f2e`

Reply:
that's unfortunate 😕 thanks for letting me know! I assume that's why it hasn't been able to be implemented for so long:(